gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/iceberg/IcebergTable.java

```diff
@@ -19,6 +19,8 @@
 
 import java.io.IOException;
 import java.net.URI;
+import java.nio.file.Path;
```

Comment:
Did you mean Hadoop Path ?

Reply:
nope... I wanted to resolve `Path` to the absolute path which is impl. in `java.nio.file.Path`. If we want to use `org.apache.hadoop.fs.Path` I will have to use `makeQualified(URI defaultUri, Path workingDir )`. Let me know the concern behind not using the current java lib and I can change it